LoopTests/ViewModels/BolusEntryViewModelTests.swift

```diff
@@ -15,9 +15,16 @@ import XCTest
 @testable import Loop
 
 class BolusEntryViewModelTests: XCTestCase {
+    override static func setUp() {
+        TimeZone.ReferenceType.default = TimeZone(abbreviation: "GMT")!
```

Comment:
Would prefer not overwriting the system timezone. All the schedules have associated timezone information that can be explicitly controlled in tests, and ideally we'd be mocking "now" in our tests to control the result of any lookups in those schedules. Is there some code that has some usage of Date() that is too deep to mock right now?

Reply:
It restores it in `tearDown()`, so why is this an issue?

The only reason this was needed was to fix a test that checks a date-formatting issue.  There's no way to mock that.

Reply:
Oh, I see it's about date formatting. Looks like the usage is pretty shallow; you could have BolusEntryViewModel accept a TimeZone for configuring its DateFormatter. Thats an option. Overriding the system timezone seems like a big hammer, and I wonder if we'll want to parallelize testing and this kind of code might bite us someday. But I don't see it causing immediate problems now.

Reply:
@ps2 you are right this is a big hammer.  (BTW: IIRC, I thought parallelized testing launched parallel simulators, so affecting the time zone of one shouldn't affect others, but I could be wrong, I haven't tested that).
I'll see if I can make this a bit more surgical in a later PR.

Reply:
@rickpasetto I typed up my note about DateFormatters accepting timezones before I read your statement about there being no way to mock.  Hopefully the above makes it clear that there *is* a way to mock this particular type of timezone usage: https://developer.apple.com/documentation/foundation/dateformatter/1411406-timezone

Reply:
Also, since xcode 10, parallel testing on a simulator has been an option. We don't have it turned on; just never tried; might be worth trying to speed things up, though.